cpu/sensors.py

```diff
@@ -43,7 +43,7 @@ def check_errors(cmd):
         Checks any errors in command result
         """
         errs = []
-        s_output = process.run(cmd, ignore_status=True).stderr
+        s_output = str(process.run(cmd, ignore_status=True).stderr)
```

Comment:
This can be `process.system_output().decode('utf-8')`

Reply:
@narasimhan-v i tried but that not worked so tried this way